index.js

```diff
@@ -180,17 +196,11 @@ class Logger {
     }
     //if message is an error, turn it into a pretty object because Errors aren't json.stringifiable
     if (message instanceof Error) {
-      message = serialize(message);
-      if (tags.indexOf('error') < 0 && options.addErrorTagToErrors) {
-        tags.push('error');
-      }
+      message = this.serialize(tags, message, options);
     } else if (typeof message === 'object') {
       Object.keys(message).forEach(key => {
```

Comment:
can we move this inside the serialize and have it call itself?  we are going to add more serialize logic - even if its not an error